tests/__snapshots__/create-action.test.js.snap

```diff
@@ -0,0 +1,55 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`creates project with labels passed to Dockerfile from questionnaire: Dockerfile 1`] = `
+# Use the latest version of Node.js
+#
+# You may prefer the full image:
+# FROM node
+#
+# or even an alpine image (a smaller, faster, less-feature-complete image):
+# FROM node:alpine
+#
+# You can specify a version:
+# FROM node:10-slim
+FROM node:slim
+
+# Labels for GitHub to read your action
+LABEL "com.github.actions.name"="My Project Name"
+LABEL "com.github.actions.description"="A cool project"
+# Here are all of the available icons: https://feathericons.com/
+LABEL "com.github.actions.icon"="anchor"
+# And all of the available colors: https://developer.github.com/actions/creating-github-actions/creating-a-docker-container/#label
+LABEL "com.github.actions.color"="blue"
+
+# Copy the package.json and package-lock.json
+COPY package*.json ./
+
+# Install dependencies
+RUN npm ci
+
+# Copy the rest of your action's code
+COPY . .
+
+# Run \`node /entrypoint.js\`
+ENTRYPOINT ["node", "/entrypoint.js"]
+
+`;
+
+exports[`creates project with labels passed to Dockerfile from questionnaire: entrypoint.js 1`] = `
+const { Toolkit } = require('actions-toolkit')
+const tools = new Toolkit()
+
+console.log(tools.arguments)
+
+`;
+
+exports[`creates project with labels passed to Dockerfile from questionnaire: package.json 1`] = `
+{
+  "name": "__tmp/my-project-name",
```

Comment:
Not sure this is desired and might be an implementation bug? Check out this test - essentially it runs:

```
npx actions-toolkit __tmp/my-project-name
```

Which would create a folder a couple of levels deep. Might be a bug or perhaps a side-effect of my test implementation and would love another set of eyes on this. 👀 ❤️

Reply:
Interesting... so when I try running that, it gives me a "`__tmp` doesn't exist" when it tries to make the `my-project-name` dir (which makes sense).

That said, I think its fine? Thanks for calling it out 🙏